examples/PSQL/README.md

```diff
@@ -0,0 +1,6 @@
+# Example Postgres SQL configuration files
+This folder contains example of a database connection to Postgres Versions 8.2 - 9.4 which are not currently supported in ArcGIS insights-connector-guide/wiki/Create-a-connector-type-bundle
+These configuration yaml files were tested using Postgres JDBC Driver https://jdbc.postgresql.org/download/postgresql-42.2.19.jar
```

Comment:
These configuration files were tested using the following PostgreSQL JDBC driver: https://jdbc.postgresql.org/download/postgresql-42.2.19.jar